client/me/billing-history/receipt.jsx

```diff
@@ -138,6 +139,7 @@ const BillingReceipt = React.createClass( {
 		return (
 			<div>
 				<DocumentHead title={ translate( 'Billing History' ) } />
+				<PageViewTracker path="/me/purchases/billing/:receiptId" title="Me > Billing History > Receipt" />
```

Comment:
@yoavf I don't think so - this is for analytics purposes, and we don't translate those. Feel free to refer to the usage of `<PageViewTracker />` within Calypso to be sure about that.